README.md

```diff
@@ -372,14 +390,15 @@ void GREENTEA_SETUP(const int timeout, const char *host_test_name) {
 ```
 * Corresponding log:
 ```
-[1458565465.35][SERI][INF] reset device using 'default' plugin...
-[1458565465.60][SERI][INF] wait for it...
-[1458565466.60][CONN][INF] sending preamble '2f554b1c-bbbf-4b1b-b1f0-f45493282f2c'
-[1458565466.60][SERI][TXD] mbedmbedmbedmbedmbedmbedmbedmbedmbedmbed
-[1458565466.60][SERI][TXD] {{__sync;2f554b1c-bbbf-4b1b-b1f0-f45493282f2c}}
-[1458565466.74][CONN][INF] found SYNC in stream: {{__sync;2f554b1c-bbbf-4b1b-b1f0-f45493282f2c}}, queued...
-[1458565466.74][HTST][INF] sync KV found, uuid=2f554b1c-bbbf-4b1b-b1f0-f45493282f2c, timestamp=1458565466.743000
-[1458565466.74][CONN][RXD] {{__sync;2f554b1c-bbbf-4b1b-b1f0-f45493282f2c}}
+[1530040532.96][GLRM][INF] remote resources reset...
+[1530040535.26][GLRM][TXD] mbedmbedmbedmbedmbedmbedmbedmbedmbedmbed
```

Comment:
Oh, will this still be transmitted? I thought the Mbed OS PR removed it.

Reply:
It's still there. I wanted to get rid of it but then I thought may be it was due to some legacy stuff.